pkg/resources/share.go

```diff
@@ -164,7 +165,7 @@ func ReadShare(d *schema.ResourceData, meta interface{}) error {
 	if err != nil {
 		return fmt.Errorf("error reading share (%v) err = %w", d.Id(), err)
 	}
-	if err := d.Set(FullyQualifiedNameAttributeName, id.FullyQualifiedName()); err != nil {
+	if err := d.Set(FullyQualifiedNameAttributeName, share.ID().FullyQualifiedName()); err != nil {
```

Comment:
Nit: why? For other resources we use id.FullyQualifiedName() and I think we can be consistent here.

Reply:
Changed